docs_src/microservices/general/index.md

```diff
@@ -0,0 +1,27 @@
+# Cross Cutting Concerns
+
+## Event Tagging
+
+In an edge solution, it is likely that several instances of EdgeX are all sending edge data into a central location (enterprise system, cloud provider, etc.)
+
+![image](MultipleInstances.png)
+
+In these circumstances, it will be critical to associate the data to its origin.  That origin could be specified by the GPS location of the sensor, the name or identification of the sensor, the name or identification of some edge gateway that originally collected the data, or many other means.
+
+EdgeX provides the means to “tag” the event data from any point in the system.  The Event object has a `Tags` property which is a key/value pair map that allows any service that creates or otherwise handles events to add custom information to the Event in order to help identify its origin or otherwise label it before it is sent to the [north side](../../general/Definitions.md#south-and-north-side).
+
+For example, a device service could populate the `Tags` property with latitude and longitude key/value pairs of the physical location of the sensor when the Event is created to send sensed information to Core Data.
+
+When the Event gets to an [Application Service](../application/ApplicationServices.md), the service has an optional function (defined by `Writable.Pipeline.Functions.AddTags` in configuration) that will add a default key/value pair to the Event `Tags`.  The key and value for the default tag are provided in configuration (as shown by the example below).  Multiple tags can be provide separated by commas.
```

Comment:
How about `additional` instead of `default`. I.e. `that will add additional key/value pair(s) to the Event`.
The change default in following text to `additional`.